data/style/style.css

```diff
@@ -2,6 +2,7 @@ window {
     padding: 12px 20px;
     border-radius: 999px;
     border: none;
+    background: alpha(@theme_bg_color, 0.8);
```

Comment:
The Window is marked as a OSD window and the GTK theme should automatically theme the background. I'm not sure if this is necessary. Any input?